dds/DCPS/MultiTopicDataReaderBase.cpp

```diff
@@ -284,14 +284,14 @@ void MultiTopicDataReaderBase::data_available(DDS::DataReader_ptr reader)
               gen.info_[i].instance_state);
           }
         } else {
-          ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) MultiTopicDataReaderBase::data_available:"
-            " failed to obtain DataReaderImpl.")));
+          ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) MultiTopicDataReaderBase::data_available:")
+            ACE_TEXT(" failed to obtain DataReaderImpl.")));
         }
       }
     }
   } catch (const std::runtime_error& e) {
     if (OpenDDS::DCPS::DCPS_debug_level) {
-      ACE_ERROR((LM_ERROR, "(%P|%t) MultiTopicDataReaderBase::data_available: "
+      ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) MultiTopicDataReaderBase::data_available: ")
         "%C", e.what()));
```

Comment:
The wchar problem is still present on this line.  I suggest making the %C part of line 294 instead of 295.